ledger/src/shred.rs

```diff
@@ -116,6 +119,20 @@ const SIZE_OF_SIGNATURE: usize = SIGNATURE_BYTES;
 // a &[Entry].
 pub const DATA_SHREDS_PER_FEC_BLOCK: usize = 32;
 
+// Statically compute the typical data batch size assuming:
+// 1. Maximum number of merkle proofs
+// 2. Merkles are chained
+// 3. No retransmit signature
+lazy_static! {
+    pub static ref DATA_SHRED_BYTES_PER_BATCH_TYPICAL: u64 = {
+        let capacity = ShredData::capacity(Some((MAX_MERKLE_PROOF_ENTRIES, true, false)))
+            .expect("Failed to get shred capacity");
+        (DATA_SHREDS_PER_FEC_BLOCK * capacity)
+            .try_into()
```

Comment:
as u64? This can basically never fail on conversion.

Reply:
done